godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -30,6 +67,138 @@ class TractViewController: BaseViewController {
         }
     }
     
+    fileprivate func initializeView() {
+        let navigationBarFrame = navigationController!.navigationBar.frame
+        let startingPoint = navigationBarFrame.origin.y + navigationBarFrame.size.height
+        
+        let width = self.view.frame.size.width
+        let height = self.view.frame.size.height - startingPoint
+        self.containerView.frame = CGRect(x: 0.0, y: startingPoint, width: width, height: height)
+        self.view.addSubview(self.containerView)
+        
+        buildPages(width, height)
+    }
+    
+    fileprivate func buildPages(_ width: CGFloat, _ height: CGFloat) {
```

Comment:
I don't think width and height should be suppressed here by adding the `_` because it is not implicit from the function name what parameters the caller should pass in. Can you remove the underscores and change it so that the caller specifies `width:` and `height:` . 

I know it will be redundant, but I think it would prevent confusion later if variable names change.